client/src/app/interfaces/event.interface.ts

```diff
@@ -18,10 +34,28 @@ export class IEventSchema {
     endDate: Date;
     location: IEventLocation;
     description?: string;
+    transportBus?: boolean;
+    transportCar?: boolean;
+    accommodation?: boolean;
     owner: string;
     editors: Array<string>;
     participants: Array<string>;
     limit?: Number;
+
+    constructor() {
+        this.name = '';
```

Comment:
Was there a reason to initialize these properties?
Or did ts complain about contructing them?

Reply:
probably there was since I added this 😄 but I guess I can remove it again